main.go

```diff
@@ -23,38 +30,79 @@ var postgresFunctions = []runtime.Transform{
 	},
 }
 
+var (
+	Network  string = "unix"
+	LogLevel int    = 1
+)
+
+type server struct {
+	pb.UnimplementedContainerizedFunctionRunnerServiceServer
+	ctx context.Context
```

Comment:
We should not store context in a struct. Check out [this](https://go.dev/blog/context-and-structs) article. Kudos to @glrf